### PR TITLE
Publish C# 11 speclets

### LINKED ISSUE #31408: Publish C# 11 speclets
See dotnet/csharplang#6451

The proposals for features delivered in C# 11 have been moved to a subfolder. It's time to publish them.

Tasks:

- [ ] Publish C# 11 folder.
- [ ] Update folder based meta-data for C# 11 proposals
- [ ] Update TOC for C# 11 proposals
- [ ] Add links to C# 11 speclets from pages in the language reference that have been updated based on C# 11 features.

### LINKED ISSUE #31458: Direkt Link to latest versions.
[Enter feedback here]
The Link https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/ will open C#10 not C#11. Let it open always the latest version.
The same is valid for  https://learn.microsoft.com/en-us/dotnet/core/whats-new/ which opens .NET Core 3.1 instead of .NET 6.0

BR
Ronny

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 1f94e527-6240-7b69-9d98-da8405790b1c
* Version Independent ID: a3d4d1bb-cfb0-3306-3ce7-5977fb199255
* Content: [What's new in C# 11 - C# Guide](https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-11)
* Content Source: [docs/csharp/whats-new/csharp-11.md](https://github.com/dotnet/docs/blob/main/docs/csharp/whats-new/csharp-11.md)
* Product: **dotnet-csharp**
* Technology: **csharp-whats-new**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by BillWagner**
Add the C# 11 speclets to the C# TOC and the list of published specifications.

Fixes #31408 
Fixes #31458

### CODE REVIEW

File: docs/core/compatibility/core-libraries/6.0/static-abstract-interface-methods.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: "Breaking change: Static abstract members in interfaces"
 description: Learn about the .NET 6 breaking change where `static` interface members can now be marked `abstract`.
 ms.date: 09/08/2021
 ---
-# Static abstract members in interfaces
+# Static abstract members declared in interfaces
```
Comment (BillWagner) [inline docs/core/compatibility/core-libraries/6.0/static-abstract-interface-methods.md:6]: ping @gewarren Is this change OK? I need to update the title for this PR to get a clean build. The feature spec in the C# 11 folder has the same H1.

File: docs/csharp/language-reference/builtin-types/integral-numeric-types.md [hunk 2]
```diff
@@ -152,6 +152,8 @@ For more information, see the following sections of the [C# language specificati
 
 - [Integral types](~/_csharpstandard/standard/types.md#836-integral-types)
 - [Integer literals](~/_csharpstandard/standard/lexical-structure.md#6453-integer-literals)
+- [C# 9 - Natural sized integral types](~/_csharplang/proposals/csharp-9.0/native-integers.md)
```
Comment (tdykstra) [inline docs/csharp/language-reference/builtin-types/integral-numeric-types.md:155]:
```suggestion
- [C# 9 - Native sized integral types](~/_csharplang/proposals/csharp-9.0/native-integers.md)
```

File: docs/csharp/language-reference/tokens/interpolated.md [hunk 3]
```diff
@@ -109,7 +109,7 @@ Beginning with C# 10, when an interpolated string is used, the compiler checks i
 
 ## C# language specification
 
-For more information, see the [Interpolated strings](~/_csharpstandard/standard/expressions.md#1173-interpolated-string-expressions) section of the [C# language specification](~/_csharpstandard/standard/README.md).
+For more information, see the [Interpolated strings](~/_csharpstandard/standard/expressions.md#1173-interpolated-string-expressions) section of the [C# language specification](~/_csharpstandard/standard/README.md), the [C# 11 - Raw string literals](~/_csharplang/proposals/csharp-11.0/raw-string-literal.md) feature specification and the [C# 11 - Newlines in string interpolations](~/_csharplang/proposals/csharp-11.0/new-line-in-interpolation.md).
```
Comment (tdykstra) [inline docs/csharp/language-reference/tokens/interpolated.md:112]:
```suggestion
For more information, see the [Interpolated strings](~/_csharpstandard/standard/expressions.md#1173-interpolated-string-expressions) section of the [C# language specification](~/_csharpstandard/standard/README.md), the [C# 11 - Raw string literals](~/_csharplang/proposals/csharp-11.0/raw-string-literal.md) feature specification and the [C# 11 - Newlines in string interpolations](~/_csharplang/proposals/csharp-11.0/new-line-in-interpolation.md) feature specification.
```

**Review by tdykstra — APPROVED** [review]
LGTM. just a couple of typos.

File: .openpublishing.redirection.core.json [hunk 4]
```diff
@@ -1105,7 +1105,7 @@
         },
         {
             "source_path_from_root": "/docs/core/whats-new/index.md",
-            "redirect_url": "/dotnet/core/whats-new/dotnet-core-3-1",
+            "redirect_url": "/dotnet/core/whats-new/dotnet-6",
             "ms.custom": "updateeachrelease"
```
Comment (mnMomma) [inline .openpublishing.redirection.core.json:1109]: Update

**Pull request merged**